doc/system_requirements.md

```diff
@@ -206,16 +206,17 @@ Covers:
 Needs: dsn
 
 #### Shortened Docker Image References
-`req~shortened-docker-image-references~1`
+`req~shortened-docker-image-references~2`
 
+Users can reference an Exasol Docker image image names of the following kind:
```

Comment:
Suggest:
Users can reference an Exasol Docker image names of the following kind:
or
Users can reference an "Exasol Docker image" image names of the following kind:

Reply:
Good point. Remove duplication.